WalletWasabi/CoinJoin/Coordinator/Rounds/CoordinatorRound.cs

```diff
@@ -1199,11 +1207,21 @@ public async Task<IEnumerable<Alice>> RemoveAlicesIfAnInputRefusedByMempoolAsync
 						Logger.LogInfo($"Mempool acceptance failed: {result.reason ?? "no reason"}.");
 					}
 				}
-			}
 
-			foreach (var alice in alicesRemoved)
-			{
-				Logger.LogInfo($"Round ({RoundId}): Alice ({alice.UniqueId}) removed.");
+				foreach (var alice in alicesRemoved)
+				{
+					Logger.LogInfo($"Round ({RoundId}): Alice ({alice.UniqueId}) removed.");
+				}
+
+				if (alicesRemoved.Count == 0)
+				{
+					Logger.LogInfo($"Overall Mempool acceptance failed, but could not found guilty Alice: {resultAll.rejectReason ?? "no reason"}.");
+				}
+				else
+				{
+					var utxosToBan = alicesRemoved.SelectMany(alice => alice.Inputs).Select(x => x.Outpoint);
+					await UtxoReferee.BanUtxosAsync(1, DateTimeOffset.UtcNow, forceNoted: false, RoundId, utxosToBan.ToArray()).ConfigureAwait(false);
```

Comment:
@nopara73 @lontivero How about banning the guilty Alices here? If we are not doing that they can come back later and make this fail. However, we are checking them in an earlier stage:

https://github.com/zkSNACKs/WalletWasabi/pull/3662/files#diff-623abb936502e8bb38181becd620794bR238

Reply:
If we decide to ban we have to consider decreasing the ban time.

Reply:
I have removed banning. It is a too complex change, to start with.